install.sh

```diff
@@ -239,6 +239,7 @@ function setup_liqo_version() {
 		warn "[PRE-FLIGHT] [DOWNLOAD]" "An unreleased version of Liqo is going to be downloaded"
 		LIQO_IMAGE_VERSION=$(get_repo_master_commit ${LIQO_REPO}) ||
 			fatal "[PRE-FLIGHT] [DOWNLOAD]" "Failed to retrieve the latest commit of the master branch"
+    LIQO_SUFFIX="-ci"
```

Comment:
Please, fix the indentation. The installer uses tabs for the indentation since spaces do not work in the `cat <<EOF` blocks